src/Components/Actions/Macros.php

```diff
@@ -194,5 +194,27 @@ public static function boot(): void
 
             return $this;
         });
+
+        Button::macro('confirm', function (?string $message = null) {
+            $this->dynamicProperties['confirm'] = [
+                'component' => 'button',
+                'attribute' => 'wire:confirm',
+                'value'     => $message ?? trans('livewire-powergrid::datatable.buttons-macros.confirm.message'),
+            ];
+
+            return $this;
+        });
+
+        Button::macro('confirmPrompt', function (?string $message = null, string $confirm_value = 'Confirm') {
+            $message = $message ?? trans('livewire-powergrid::datatable.buttons-macros.confirm-prompt.message', ['confirm_value' => $confirm_value]);
+            $confirm_value = trim($confirm_value);
+            $this->dynamicProperties['confirmPrompt'] = [
+                'component' => 'button',
+                'attribute' => 'wire:confirm.prompt',
+                'value'     => "$message |$confirm_value",
```

Comment:
Use camelCase

Reply:
camelCase for $confirm_value ?

so did I also need to change the translations placeholder for `:confirm_value` to use **camelCase** `:confirmValue`.